src/mock-dnas/hpos.js

```diff
@@ -63,6 +63,7 @@ const data = {
     // Forces HoloPortOS upgrade. - Returns Status Code ONLY (200 if OK/Successful, 400 if Bad Request, or 401 if Unauthorized).
     // TODO: Talk to HCen to see if hydra/hpos will return new Status Obj upon successful update.
     '/v1/upgrade': () => {
+      console.log('Code 200: You have now updated your HPOS Version.')
```

Comment:
Did you mean to leave this log in? I feel ambivalent about it, leaning towards removing it. We already have logs that tell us which dna endpoints are being called.

Reply:
I could go either way, but do have a slight preference to leave it in as a quick visual/semantic reminder.